### DISCUSSION TITLE
Can FDS be used for machine learning databases,thanks

### DISCUSSION THREAD

**Discussion by chenlinjie1213** [firemodels/fds#11680] [General]
Dear Friends, I would like to ask if you have some databases related to FDS? I would like to carry out some machine learning training based on the data from FDS simulation results, thanks.

Thank you.

**Comment by johodges** [comment]
The largest publicly available repo of FDS simulations is probably the validation output repo:

https://github.com/firemodels/out

What specific task are you hoping to use machine learning to address?

**Comment by rmcdermo** [comment]
Select output used for FDS validation are stored in the [firemodels/out](https://github.com/firemodels/out) repository.
